### ISSUE TITLE
"crop": false, "crop_aspect": "random" aspect ratio bucket should have sane ratio minima/maxima

### ISSUE THREAD

**Issue by AmericanPresidentJimmyCarter** [bghira/SimpleTuner#1113]
`Processing bucket 74.0:   0%| ` probably should not happen. I don't think such an image works with the positional embeddings of most models. Consider a `ratio_min` and `ratio_max` option in the dataset configurations.

**Comment by bghira** [comment]
well, if crop is false then crop_aspect=random does not apply

**Comment by AmericanPresidentJimmyCarter** [comment]
OK, edited title.

**Comment by bghira** [comment]
is your expectation that it functions like `minimum_image_size`  and simply exclude the image from processing?

**Comment by AmericanPresidentJimmyCarter** [comment]
Yeah, I implemented it locally like that. If you exclude it at the image processing step it's easy.

**Comment by bghira** [comment]
ah, ok. initially i thought you wanted it just placed into a different min or max bucket
